config/routes.rb

```diff
@@ -11,6 +11,7 @@
   get 'evidence/:id/income', to: 'evidence#income', as: :evidence_income
   post 'evidence/:id/income_save', to: 'evidence#income_save', as: :evidence_income_save
   get 'evidence/:id/result', to: 'evidence#result', as: :evidence_result
+  get 'evidence/:id/summary', to: 'evidence#summary', as: :evidence_summary
```

Comment:
What checks this route?

I ask only because all the other routes for this controller have the route specs.